### PR TITLE
Commerce 5685

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-5685

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/render/view.jsp [hunk 1]
```diff
@@ -31,6 +31,12 @@ CSDiagramCPTypeDisplayContext csDiagramCPTypeDisplayContext = (CSDiagramCPTypeDi
 	props='<%=
 		HashMapBuilder.<String, Object>put(
 			"enablePanZoom", true
+		).put(
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/render/view.jsp:34]: take this from CommerceContext it is a request

**Comment by andrea-ale-sbarra** [comment]
I need to reorganize the commit and I had to remove the AddToCart component because of some inconsistency with the loader. Unfortunately I cannot continue work on this at the moment.

**Comment by andrea-ale-sbarra** [comment]
This will be reworked after the refactor of Diagram components @FabioDiegoMastrorilli @garofalog @ivicac

**Pull request closed without merging**